sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryAvroUtilsTest.java

```diff
@@ -99,28 +103,30 @@ public void testConvertGenericRecordToTableRow() throws Exception {
     // type. AvroCoder can't apply logical types to Schemas directly, so we need to get the
     // Schema for the Bird class defined below, then replace the field used to test NUMERIC with
     // a field that has the appropriate Schema.
-    BigDecimal birthdayMoney = new BigDecimal("123456789.123456789");
-    Schema birthdayMoneySchema = Schema.create(Type.BYTES);
-    LogicalType birthdayMoneyLogicalType =
-        LogicalTypes.decimal(birthdayMoney.precision(), birthdayMoney.scale());
+    BigDecimal bigDecimal =
+        new BigDecimal(
+            "578960446186580977117854925043439539266.34992332820282019728792003956564819967");
```

Comment:
If you're going to use this for both NUMERIC and BIGNUMERIC tests, can you keep the value within the supported range for both? I think the prior value `123456789.123456789` would work fine.

Reply:
Decided to use separate numbers for both the NUMERIC and BIGNUMERIC tests.